tests/integration/test_integration.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+
+import logging
+from pathlib import Path
+
+import pytest
+import requests
+import yaml
+from alertmanager import Alertmanager
+from pytest_operator.plugin import OpsTest  # type: ignore[import]  # noqa: F401
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+ALERTMANAGER_CONFIGURER_APP_NAME = METADATA["name"]
+ALERTMANAGER_APP_NAME = "alertmanager-k8s"
+WAIT_FOR_STATUS_TIMEOUT = 1 * 60
+DUMMY_HTTP_SERVER_PORT = 80
+
+
+class TestAlertmanagerConfigurerOperatorCharm:
+    @pytest.fixture(scope="module")
+    @pytest.mark.abort_on_fail
+    async def setup(self, ops_test: OpsTest):
+        await ops_test.model.set_config({"update-status-hook-interval": "2s"})
```

Comment:
Out of curiosity, why do we need to set this?

Reply:
To speed up tests. It makes framework react faster to the changes.